Tiamat.py

```diff
@@ -21,15 +21,13 @@ async def async_post(url, payload, headers):
 class Tiamat:
     """TODO."""
 
-    def __init__(self, editwin, run_in_idle=True):
```

Comment:
Removing this feature ideally should have been a reversion, i.e. `git revert`, so that the changes were atomic.